projects/liondex/index.js

```diff
@@ -0,0 +1,18 @@
+const { staking } = require('../helper/staking')
+const { sumTokens2,sumTokensExport } = require('../helper/unwrapLPs')
+
+const treasure = "0x7fca3bf8adc4e143bd789aecda36c0ce34f1d75b";
```

Comment:
hi, this is team multisig? we can count it under treasury metric but not as tvl, sorry.

Reply:
this is the protocol treasury, not for team.   all funds  used for protocol development.

Reply:
hi, is that ok ?  @g1nt0ki

Reply:
hi @LionDEXDev , yes but users can never withdraw this amount right? thats why I think it should be counted as treasury not tvl

Reply:
thanks for your reply, bro. the protocol treasury belong to the community, so the use of funds will be decided through community voting, such as adding liquidity, repurchasing tokens, even withdrawals, etc.   Does this make sense？